packages/kbn-shared-ux-components/src/page_template/solution_nav/solution_nav_collapse_button.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import './solution_nav_collapse_button.scss';
+
+import React from 'react';
+import classNames from 'classnames';
+
+import { EuiButtonIcon, EuiButtonIconPropsForButton } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+export type KibanaPageTemplateSolutionNavCollapseButtonProps =
+  Partial<EuiButtonIconPropsForButton> & {
+    /**
+     * Boolean state of current collapsed status
+     */
+    collapsed: boolean;
```

Comment:
nit: Why change from `isCollapsed`?

Reply:
I thought this was our preferred way of naming props? Without `is`?

Reply:
gotcha! changed that